src/app/shared/main-navigation/mat-main-nav/status-bar/status-bar.component.ts

```diff
@@ -22,7 +22,9 @@ export class StatusBarComponent implements OnInit {
   }
 
   setUsername(): void {
-    this.username = this.authService.userName;
+    this.username = this.authService.userName
```

Comment:
I guess we always get the username now(email) and very less scope for fullName to return.

Reply:
no "custom:preferred_username" when login with cognito user. blank